docs/static/include/pluginbody.asciidoc

```diff
@@ -779,6 +779,36 @@ will be discussed further in the testing section of this document.
 Another kind of external dependency is on jar files.  This will be described
 in the "Add a `gemspec` file" section.
 
+===== Dependencies on deprecated features
+
+Sometimes a plugin relies on a feature from another plugin. And sometimes those
```

Comment:
🤔 The purpose of the deprecation logger is to allow plugins to warn of deprecations in a place that isn't flooded by other log messages, so that they can upgrade with confidence (e.g., when a plugin is going to stop supporting an option in a future release, or is going to change the default value of an option, etc.). It's not specifically about dependencies.

~~~ asciidoc
===== Deprecated features

As a plugin evolves, sometimes it becomes clear that a particular option or feature
is no longer serving the intended purpose, and the developer may want to _deprecate_
its usage, warning users away from its usage so they aren't caught by surprise when
it is eventually removed in a later release.
{ls} 7.6 introduced a _deprecation logger_ to make handling those situations easier, and an 
https://github.com/logstash-plugins/logstash-mixin-deprecation_logger_support[adapter]
can be used to ensure your plugin can use the deprecation logger while still supporting
older versions of {ls}.
~~~